### PR TITLE
python: add manual tracing docs

### PR DESCRIPTION
**Pull request by Kyle-Verhoog**
### What does this PR do?

Updates and adds to our existing manual instrumentation documentation.

### CODE REVIEW

File: content/en/tracing/manual_instrumentation/python.md [hunk 1]
```diff
@@ -21,63 +17,87 @@ If you aren't using supported library instrumentation (see [library compatibilit
...
+    span = tracer.trace("sandwich.create")
+    ingredients = get_ingredients()
+    sandwich = assemble_sandwich(ingredients)
+    span.finish()  # remember to finish the span
 ```
 
-API details of the decorator can be found here:
+API details of the decorator can be found below:
 
 - [`ddtrace.Tracer.trace`][2]
```
Comment (sarina-dd) [inline content/en/tracing/manual_instrumentation/python.md:102]:
```suggestion
```

File: content/en/tracing/manual_instrumentation/python.md [hunk 2]
```diff
@@ -21,63 +17,87 @@ If you aren't using supported library instrumentation (see [library compatibilit
...
 - [`ddtrace.Span.finish`][3]
```
Comment (sarina-dd) [inline content/en/tracing/manual_instrumentation/python.md:103]:
```suggestion
```

File: content/en/tracing/manual_instrumentation/python.md [hunk 3]
```diff
@@ -21,63 +17,87 @@ If you aren't using supported library instrumentation (see [library compatibilit
...
-  span.finish()
+def make_sandwich_request(request):
+    span = tracer.trace("sandwich.create")
+    ingredients = get_ingredients()
+    sandwich = assemble_sandwich(ingredients)
+    span.finish()  # remember to finish the span
 ```
 
-API details of the decorator can be found here:
+API details of the decorator can be found below:
```
Comment (sarina-dd) [inline content/en/tracing/manual_instrumentation/python.md:100]:
```suggestion
API details of the decorator can be found in the `ddtrace.Tracer.trace` [documentation][2] or the `ddtrace.Span.finish`[documentation][3].
```

File: content/en/tracing/manual_instrumentation/python.md [hunk 4]
```diff
@@ -21,63 +17,87 @@ If you aren't using supported library instrumentation (see [library compatibilit
 
 You may also want to extend the functionality of the `ddtrace` library or gain finer control over instrumenting your application. Several techniques are provided by the library to accomplish this.
 
-The following examples use the global tracer object which can be imported via:
+## Creating spans
+
+The `ddtrace` library will create spans automatically with `ddtrace-run` for [many libraries and frameworks][1]. However, you may want to gain visibility into your own code and this is achieved using spans.
```
Comment (sarina-dd) [inline content/en/tracing/manual_instrumentation/python.md:22]:
```suggestion
The `ddtrace` library creates spans automatically with `ddtrace-run` for [many libraries and frameworks][1]. However, you may want to gain visibility into your own code and this is achieved by using spans.
```

**Comment by apigirl** [comment]
hi! we recently updated our S3 strategy in staging, so I had to merge master into this PR for the staging link to continue to work. If you're working locally, you'll just need to pull in my changes.

File: content/en/tracing/manual_instrumentation/python.md [hunk 5]
```diff
@@ -21,63 +17,87 @@ If you aren't using supported library instrumentation (see [library compatibilit
...
+
+For instance, within your make sandwich web request you may perform several operations, `get_ingredients()` and `assemble_sandwich()`  which would be useful to measure.
```
Comment (sarina-dd) [inline content/en/tracing/manual_instrumentation/python.md:24]:
```suggestion
Within your web request (for example, `make_sandwich_request`), you may perform several operations, like `get_ingredients()` and `assemble_sandwich()`, which are useful to measure.
```

File: content/en/tracing/manual_instrumentation/python.md [hunk 6]
```diff
@@ -89,8 +109,141 @@ API details of the decorator can be found here:
...
+span.error = 1
+span.finish()
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+
+## OpenTracing
+
+OpenTracing support is included in the `ddtrace` package. Use `pip` to install the required `opentracing` package :
```
Comment (sarina-dd) [inline content/en/tracing/manual_instrumentation/python.md:209]:
```suggestion
OpenTracing support is included in the `ddtrace` package. Use `pip` to install the required `opentracing` package:
```

File: content/en/tracing/manual_instrumentation/python.md [hunk 7]
```diff
@@ -89,8 +109,141 @@ API details of the decorator can be found here:
...
+  span = opentracing.tracer.start_span("<OPERATION_NAME>")
+  span.set_tag("<TAG_KEY>", "<TAG_VALUE>")
+  time.sleep(0.05)
+  span.finish()
+
+init_tracer("<SERVICE_NAME>")
+my_operation()
+```
+
+The tracer can now be used just like in any other OpenTracing application. See [opentracing.io][2] for OpenTracing Python usage.
```
Comment (sarina-dd) [inline content/en/tracing/manual_instrumentation/python.md:241]:
```suggestion
The tracer can now be used like in any other OpenTracing application. See [opentracing.io][2] for OpenTracing Python usage.
```

**Pull request merged**